index.html

```diff
@@ -138,7 +138,4 @@
 </div>
 {% endcapture %}
 
-{% include sub_layouts/default_layout.html %}
-
-<script src="{{ root_path }}/js/lib/highcharts.js"></script>
```

Comment:
@pbadger  highcharts is only used on the homepage. I'm reluctant to include it everywhere since it's like 59k.  Thoughts?  Everything else made sense to me.